_includes/menu.html

```diff
@@ -7,8 +7,8 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="../">
-        <span>Intercity.io</span>
+      <a class="navbar-brand" href="/">
+        <img src="/images/logo.svg">
```

Comment:
This would be the result:

![schermafbeelding 2016-09-05 om 14 48 01](https://cloud.githubusercontent.com/assets/42268/18248326/c4c79c68-7377-11e6-8706-df922624b0dd.png)